src/main/java/com/aws/greengrass/tes/TokenExchangeService.java

```diff
@@ -37,6 +38,9 @@
     private static final String TES_CONFIG_ERROR_STR = "%s parameter is either empty or not configured for TES";
     // randomly choose a port
     private static final int DEFAULT_PORT = 0;
+    // Use a time in the far future so that when we use withNewerValue, the configuration change will always happen,
+    // no matter what the current system clock is (as long as the clock isn't ahead of 3000)
+    private static final long FUTURE = Instant.parse("3000-01-01T00:00:00Z").toEpochMilli();
```

Comment:
Can you break this into a separate commit? I'd prefer having a commit message that adds a little more context

Reply:
Different commit, same PR, done.